src/System.CommandLine/CommandLineConfiguration.cs

```diff
@@ -19,108 +18,76 @@ public class CommandLineConfiguration
         private TextWriter? _output, _error;
 
         /// <summary>
-        /// A delegate that will be called when an exception is thrown by a command handler.
+        /// Initializes a new instance of the <see cref="CommandLineConfiguration"/> class.
         /// </summary>
-        internal readonly Func<Exception, InvocationContext, int>? ExceptionHandler;
+        /// <param name="rootCommand">The root command for the parser.</param>
+        public CommandLineConfiguration(Command rootCommand)
+        {
+            RootCommand = rootCommand ?? throw new ArgumentNullException(nameof(rootCommand));
+            Directives = new()
+            {
+                new SuggestDirective()
```

Comment:
I was not sure which directives should be enabled by default, it's an open question.

I have not enabled all of them, as it would cause a perf hit for the users.